src/nfhl/config.py

```diff
@@ -0,0 +1,58 @@
+"""
+config.py: Configuration values. Secrets to be handled with Secrets Manager
+"""
+
+import logging
+import socket
+
+SKID_NAME = "nfhl_skid"
+
+AGOL_ORG = "https://utah-em.maps.arcgis.com"
+SENDGRID_SETTINGS = {  #: Settings for SendGridHandler
+    "from_address": "noreply@utah.gov",
+    "to_addresses": "jdadams@utah.gov",
```

Comment:
**question**: how long do we expect this skid to live? If it's some what permanent, I recommend updating this to ugrc-developers.